asterisk/rootfs/etc/cont-init.d/asterisk.sh

```diff
@@ -70,19 +73,47 @@ bashio::var.json \
     -template /usr/share/tempio/http.conf.gtpl \
     -out /config/asterisk/http.conf
 
-persons="$(curl -s -X GET \
-    -H "Authorization: Bearer ${SUPERVISOR_TOKEN}" \
-    -H "Content-Type: application/json" \
-    http://supervisor/core/api/states |
-    jq -c '[.[] | select(.entity_id | contains("person.")).attributes.id]')"
 
-auto_add=$(bashio::config 'auto_add')
-auto_add_secret=$(bashio::config 'auto_add_secret')
-video_support=$(bashio::config 'video_support')
+# Retrieve the list of persons from the HA API
+function get_persons() {
+    
+    # Load the base url for Home Assistant as an env variable. 
+    # If not defined (we are running as an add-on), fallback to the supervisor internal proxy URL
+    HOME_ASSISTANT_URL=${HOME_ASSISTANT_URL:-"http://supervisor/core"}
```

Comment:
```suggestion
    # Load the base url for Home Assistant as an env variable. 
    # If not defined (we are running as an add-on), fallback to the supervisor internal proxy URL
    local ha_url=${HOME_ASSISTANT_URL:-"http://supervisor/core"}
```

Should be a local variable, right?